collector/receiver/otelarrowreceiver/internal/arrow/arrow.go

```diff
@@ -94,6 +99,26 @@ func New(
 		gsettings:   gsettings,
 		netReporter: netReporter,
 	}
+
+	meter := recv.telemetry.MeterProvider.Meter(scopeName)
+	recv.recvInFlightBytes, err = meter.Int64UpDownCounter(
+		"otelarrow_receiver_in_flight_bytes",
```

Comment:
nit: we've used `otel_arrow_` as as a prefix for span names in this code, please use that here.

Reply:
Thanks for the review! Fixed